test/Microsoft.ML.Tests/Scenarios/ClusteringTests.cs

```diff
@@ -0,0 +1,122 @@
+﻿using Microsoft.ML.Data;
+using Microsoft.ML.Runtime;
+using Microsoft.ML.Runtime.Api;
+using Microsoft.ML.Trainers;
+using Microsoft.ML.Transforms;
+using System;
+using System.Collections.Generic;
+using Xunit;
+
+namespace Microsoft.ML.Scenarios
+{
+    public partial class ScenariosTests
+    {
+        [Fact(Skip = "Missing data set. See https://github.com/dotnet/machinelearning/issues/203")]
+        public void PredictNewsCluster()
+        {
+            string dataPath = GetDataPath(@"external/20newsgroups.txt");
+
+            var pipeline = new LearningPipeline();
+            pipeline.Add(new TextLoader(dataPath).CreateFrom<NewsData>(useHeader: false));
+            pipeline.Add(new CategoricalOneHotVectorizer("Label"));
```

Comment:
I think "Label" is not used in clustering unless model is being evaluated against true labels. Why CategoricalOneHotVectorizer is being applied on "Label"?